src/cc/bcc_syms.cc

```diff
@@ -13,8 +13,13 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+#ifndef _GNU_SOURCE
```

Comment:
@4ast : Do we need this? Which part is GNU source (I moved this from `bcc_proc.c`)